.github/config/rubocop_linter_action.yml

```diff
@@ -0,0 +1,67 @@
+# .github/config/rubocop_linter_action.yml
+
+# Description: The name of the check that will be created.
+# Valid Options: A reasonably sized string.
+# Default: 'RuboCop Action'
+check_name: "RuboCop Results"
+
+# Description: Versions required to run your RuboCop checks.
+# Valid options: RuboCop and any RuboCop extension, by default the latest gem version will be used. You can explicitly state that
+# (not required) or use a version number, like '1.5.1'.
+# Default:
+#   versions:
+#     - rubocop: 'latest'
+versions:
+  - rubocop
+  - rubocop-rails
+  - rubocop-minitest
+  - rubocop-performance: "1.5.1"
+  - rubocop-rspec: "1.37.0"
+  - rubocop-rootstrap
```

Comment:
I'm looking at your Gemfile and I would remove `rubocop-rootstrap` cause it's a little bit deprecated, and install `rubocop` and `rubocop-rspec` directly
![Screenshot 2021-08-12 at 18 14 54](https://user-images.githubusercontent.com/33032571/129270964-53693697-de93-46b2-946d-e78718546c81.png)

Reply:
ready